elytron/src/test/java/org/wildfly/extension/elytron/LdapTestCase.java

```diff
@@ -146,6 +147,7 @@ private static void loadResources() throws Exception {
     public static void startLdapService() throws Exception {
         loadResources();
         TestEnvironment.startLdapService();
+        Security.setProperty("keystore.PKCS12.keyProtectionAlgorithm", "PBEWithHmacSHA256AndAES_128");
```

Comment:
There's a PR that fixes `LdapTestCase` (https://github.com/wildfly/wildfly-core/pull/4577) so this change can be removed here.